docs/current/cli/979595-reference.md

```diff
@@ -113,6 +173,59 @@ Log out of Dagger Cloud:
 dagger logout
 ```
 
+## dagger mod
+
+Manage Dagger modules. By default, print the configuration of the current module in JSON format.
+
+### Usage
+
+```shell
+dagger mod [--mod string] [--focus]
+dagger mod [sub-command] [--mod string] [--focus]
+```
+
+### Sub-commands
+
+| Sub-command  | Description                                                           |
+| ------------ | --------------------------------------------------------------------- |
+| `init`       | Initialize a new Dagger module in a local directory                   |
+| `install`    | Add a new dependency to a Dagger module                              |
+| `sync`       | Synchronize a Dagger module with the latest version of its extensions |
+| `publish`    | Publish a Dagger module to the Daggerverse                            |
+
+### Options
+
+| Option               | Description                           |
+| ---------------------| --------------------------------------|
+| `--focus`            | Only show output for focused commands |
+| `-m`, `--mod string` | Path to `dagger.json` config file for the module or a directory containing that file. May be a local path or a remote Git repository |
+
+### Examples
```

Comment:
Should we also have examples for `install` and `publish`? I think at least `install` would be worth including :tada: 

What do you think about maybe splitting up the sub-commands into separate headings for each subcommand, so we have more space to document each one, e.g. for individual examples, eventually adding new flags for each one, etc?

Reply:
Added